src/Text/Pandoc/Readers/DocBook.hs

```diff
@@ -708,7 +708,7 @@ parseBlock (Elem e) =
         "para"  -> parseMixed para (elContent e)
         "formalpara" -> do
            tit <- case filterChild (named "title") e of
-                        Just t  -> (para . strong . (<> str ".")) <$>
+                        Just t  -> (para . strong . (<> str " ")) <$>
```

Comment:
Why append a space? This should probably be removed entirely, maybe like:

```suggestion
                        Just t  -> (para . strong . str) <$>
```

Reply:
perfect - I am not a haskell developer and thought my change was on the safe side.
Your suggestion is perfect. Thanx!

Reply:
hm. seems not to work - breaks tests